packages/commonwealth/client/styles/components/component_kit/new_designs/cw_text_input.scss

```diff
@@ -0,0 +1,100 @@
+@import '../../../shared';
+
+@mixin iconPositionStyles {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.MessageRow {
+  display: flex;
+
+  &.hasFeedback {
+    align-items: flex-end;
+    gap: 8px;
+    justify-content: space-between;
+
+    > * {
+      flex: 1;
+    }
+  }
+
+  .feedback-message-text.Text {
+    justify-content: flex-end;
+    margin-bottom: 4px;
+    text-align: right;
+
+    &.success {
+      color: $green-600;
+    }
+
+    &.failure {
+      color: $rorange-500;
+    }
+  }
+}
+
+.TextInput {
+  display: flex;
+  flex-direction: column;
+
+  .input-and-icon-container {
+    display: flex;
+    align-items: center; /* Center icons vertically */
```

Comment:
we do not need comments like these. It is obvious what this property does

Reply:
Addressed.